File: materialm-angular-free-main/main/src/app/Models/GameDTO.ts
export class GameDTO {
    name!:string;
    startDate!:string;
    endDate!:string;
    virtualStartDate!:string;
    startingamount!:number;
    simulationDays!:number;
    candlesPerDay!:number;
    allowstoploss!:boolean;
    allowtakeprofit!:boolean;
    allowshortpos!:boolean;
    allowlongpos!:boolean;
}

File: Pi5/src/main/java/org/example/pi5/entities/GameDTO.java
package org.example.pi5.entities;

import lombok.*;

@Getter
@Setter
@Builder
@AllArgsConstructor
@NoArgsConstructor
@ToString
public class GameDTO {
    private String name;
    private String startDate;
    private String endDate;
    private String virtualStartDate;
    private int simulationDays;
    private double startingAmount;
    private boolean Allowstoploss;
    private boolean Allowshortpos;
    private boolean Allowlongpos;
    private boolean Allowtakeprofit;
    private int candlesPerDay;
}
